sdk/provisioning/Azure.Provisioning.AppConfiguration/tests/BasicAppConfigurationTests.cs

```diff
@@ -0,0 +1,88 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Provisioning.Expressions;
+using Azure.Provisioning.Tests;
+using NUnit.Framework;
+
+namespace Azure.Provisioning.AppConfiguration.Tests;
+
+public class BasicAppConfigurationTests(bool async)
+    : ProvisioningTestBase(async /*, skipTools: true, skipLiveCalls: true /**/)
+{
+    [RecordedTest]
+    [Description("https://github.com/Azure/azure-quickstart-templates/blob/master/quickstarts/microsoft.appconfiguration/app-configuration-store-ff/main.bicep")]
+    public async Task CreateAppConfigAndFeatureFlag()
+    {
+        await using Trycep test = CreateBicepTest();
+        await test.Define(
+            ctx =>
+            {
+                BicepParameter location = BicepParameter.Create<string>(nameof(location), BicepFunction.GetResourceGroup().Location);
```

Comment:
why the resource group is a method on BicepFunction. I would expect the resource group to be a CDK resource type that I need to instantiate.

Reply:
That's the standard [`resourceGroup()` function from Bicep](https://learn.microsoft.com/en-us/azure/azure-resource-manager/bicep/bicep-functions-scope#resourcegroup) and it returns your current `ResourceGroup` instance (that's an Azure.Provisioning type derived from `Resource`).  You can also construct those instances with `new ResourceGroup(...)` like anything else.